pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -414,12 +416,6 @@ func applyUpstreamTLSSettings(cluster *v2.Cluster, tls *networking.TLSSettings)
 			Sni: tls.Sni,
 		}
 	case networking.TLSSettings_MUTUAL, networking.TLSSettings_ISTIO_MUTUAL:
-		// This is already an h2 cluster, advertise it with alpn.
-		if cluster.Http2ProtocolOptions != nil {
-			// advertising h2 ensures that h2 is used and it is not downgraded to http/1.1
-			// empty alpn usually defaults to http/1.1
-			cluster.TlsContext.CommonTlsContext.AlpnProtocols = ALPNH2Only
-		}
```

Comment:
why are you removing this?

Reply:
This is reverted in head of release-0.8 already https://github.com/istio/istio/commit/9aecafae473a1fc2349c1f17dd342e9a905a2b1f

I just merged with the head.